.circleci/config.yml

```diff
@@ -209,6 +209,11 @@ jobs:
     steps:
       - checkout
       - rust_components
+      - run:
+          name: Update some packages
+          command: |
+            apt-get update
+            apt-get install -y libssl-dev pkg-config clang lld python3-dev libpython3-dev
```

Comment:
Since we're using python-build-standalone for official builds, why do we need the python dev packages? I figured we'd want the tests to be done with python-build-standalone since that would presumably be more representative of what is run in official builds?